CLIENT/client3.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import os
+import requests
+from datetime import datetime
+from dataclasses import dataclass
+from dotenv import load_dotenv
+import sqlite3
+
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/